README.md

```diff
@@ -276,3 +276,6 @@ If you are using Manticore on an academic work, consider applying to the [Crytic
 ## Demo Video from ASE 2019
 [![Brief Manticore demo video](https://img.youtube.com/vi/o6pmBJZpKAc/1.jpg)](https://youtu.be/o6pmBJZpKAc)
 
+## Tool Integrations 
+
+- [MATE: Merged Analysis To prevent Exploits](https://github.com/GaloisInc/MATE)
```

Comment:
MATE is a large system, we should call out which subdirectories specifically use Manticore. Let's also link to the docs page(s) here https://galoisinc.github.io/MATE/dwarfcore.html and https://galoisinc.github.io/MATE/mantiserve.html